docs/src/main/asciidoc/deployment-to-openshift.adoc

```diff
@@ -0,0 +1,98 @@
+The extension supports multiple deployment options, including an OpenShift binary build strategy or Docker build strategy.
+
+* By using the Quarkus CLI
++
+You can issue the `quarkus deploy` command to enable the deployment of Quarkus applications to platforms, such as Kubernetes or OpenShift Container Platform.
++
+By using the Quarkus CLI, you can deploy your Quarkus applications without needing to manually add or remove extensions to your project.
+Instead of needing to update your project configuration file (`application.properties` file), the Quarkus CLI provides a simpler approach to deploy your Quarkus applications.
++
+You can further simplify your use of the CLI by using the `--help` flag to get more information about commands or the autocompletion feature to help when issuing commands.
++
+For more information about the Quarkus CLI, see xref:cli-tooling.adoc[Building Quarkus apps with Quarkus CLI].
+
+* By using the Maven or Gradle plugins
++
+You can use the Maven and Gradle plugins to deploy Quarkus applications by running the provided deploy commands.
+For example, if you are using Maven, issue the `mvn quarkus:deploy` command.
+If you are using Gradle, issue the `gradle deploy` command.
+
+Use the information provided here to determine the strategy to use to build and deploy your Quarkus applications on OpenShift.
+
+[#overviewbuildstrategies]
+== OpenShift Container Platform build strategies overview
+
+OpenShift binary build:: This strategy leverages the OpenShift container image framework and uses a JAR file as input to the build process, which speeds up building and deploying your application.
+As an out-of-the-box feature, the OpenShift extension is configured to use the `quarkus-container-image-openshift` extension to build the container image inside the OpenShift cluster.
+For information about the OpenShift container image, see xref:container-image.adoc#openshift[container-image-openshift].
+
+non-OpenShift build:: As an alternative to the `quarkus-container-image-openshift` approach, Quarkus also supports the following extensions for building container images.
+
+- xref:container-image.adoc#docker[Container-image-docker]
+- xref:container-image.adoc#jib[Container-image-jib]
+- xref:container-image.adoc#buildpack[Container-image-buildpack]
+
+[NOTE]
+====
+The OpenShift Container Platform Docker build strategy is the preferred non-OpenShift build strategy because it supports Quarkus applications targeted for JVM or compiled to native executables.
+However, for compatibility with earlier Quarkus versions, the default build strategy is the OpenShift binary build.
+To select the OpenShift Container Platform Docker build strategy, use the `quarkus.openshift.build-strategy` property.
+====
+
```

Comment:
Something is missing here.
The extensions mentioned above are creating the container, then you need to push it.